src/app/components/GoogleSheetsInput.svelte

```diff
@@ -0,0 +1,179 @@
+<script>
+  import worker from "../spawn-worker";
+
+  import InputField from "./InputField.svelte";
+  import Button from "./Button.svelte";
+
+  var spreadsheetId = "";
+  var error = null;
```

Comment:
You'll need to add a line to do data binding (see below):

```suggestion
  var error = null;
  let googleSheetsURL = "";
```